protocol-units/execution/suzuka/executor/src/lib.rs

```diff
@@ -0,0 +1,43 @@
+pub mod v1;
+
+pub use aptos_types::{
+    transaction::signature_verified_transaction::SignatureVerifiedTransaction,
+    block_executor::partitioner::ExecutableBlock,
+    block_executor::partitioner::ExecutableTransactions,
+    transaction::{SignedTransaction, Transaction}
+};
+pub use aptos_crypto::hash::HashValue;
+use async_channel::Sender;
+use aptos_api::runtime::Apis;
+pub use movement_execution_util::FinalityMode;
+
+
+#[tonic::async_trait]
+pub trait MonzaExecutor {
```

Comment:
If this is going to be a generalised Executor :
```suggestion
pub trait MovementExecutor {
```

Reply:
No, this is a Suzuka-specific crate.

Reply:
But the trait itself could be migrated into a common crate if it needs to be, I haven't looked into such things yet.

Reply:
Yeah, this should be Suzuka specific.